website/docs/reference/resource-configs/delimiter.md

```diff
@@ -0,0 +1,76 @@
+---
+resource_types: [seeds]
+datatype: <string>
+default_value: ","
+---
+
+## Definition
+
+An optional seed configuration used to separate values in a [seed](/docs/build/seeds) with a string you provide. The delimiter defaults to comma.
+
+
+## Usage
+Specify delimiter in your `dbt_project.yml` file to globally separate all seed values:
+
+<File name='dbt_project.yml'>
+
+```yml
+seeds:
+  +quote_columns: False
+  +delimiter: ","
+```
+
+</File>
+
+
+Or only separate values with a delimiter in the seeds/mappings directory:
```

Comment:
What is the `seeds/mappings` directory?

Reply:
That seems to go to the `Use a comma delimiter` example, not the `properties.yml` example